pkg/adapter/adapter.go

```diff
@@ -355,6 +360,26 @@ func (a *Adapter) postMessage(msg wabbit.Delivery) error {
 	return nil
 }
 
+func convertToCloudEvent(event *cloudevents.Event, msg wabbit.Delivery, a *Adapter) error {
+	if msg.MessageId() != "" {
+		event.SetID(msg.MessageId())
+	} else {
+		event.SetID(string(uuid.NewUUID()))
+	}
+
+	event.SetType(sourcesv1alpha1.RabbitmqEventType)
+	event.SetSource(sourcesv1alpha1.RabbitmqEventSource(a.config.Namespace, a.config.Name, a.config.Topic))
+	event.SetSubject(event.ID())
+	event.SetExtension("key", event.ID())
```

Comment:
@gabo1208 @benmoss I am missing some context here.. what is this "key" extension?

Reply:
This has always been here in the implementation, unclear what it was for: https://github.com/knative-sandbox/eventing-rabbitmq/pull/3/files#diff-b4e88addd0235bf02451223277489d108a173bed5b86cc9f8db75eefab9dcf24R300

Reply:
It was something that was there from the start, maybe it is time to get rid of it?